gulpfile.js

```diff
@@ -21,7 +21,7 @@ gulp.task('copy', () =>
 
 // Process the Stylus files into CSS
 gulp.task('styles', () =>
-  gulp.src('static/src/styles/**/*.styl')
+  gulp.src('static/src/styles/main.styl')
```

Comment:
Can see that this works but I don't understand why. My understanding is that `gulp.src` will output all files which match the given glob which in this case is just this one file.
I thought `changed(dest)` just filtered out its input to those files that have changed, how is it finding the changed files?